crates/aptos-protos/proto/aptos/datastream/v1/datastream.proto

```diff
@@ -57,6 +60,14 @@ message RawDatastreamResponse {
   uint32 chain_id = 3;
 }
 
+// OnChainDataSummaryRequest is a request for the current on-chain data summary.
+message OnChainDataSummaryRequest {}
+message OnChainDataSummaryResponse {
+  uint32 chain_id = 1;
+  // uint64 epoch = 2;
+}
```

Comment:
hmm what is this lol. can you add more comments for how this is different from RawDatastreamResponse?

Reply:
done. added an example for cache bootstrap.